File: src/wwwroot/app/common/models/dto/countryLocalised.ts
module Antares.Common.Models.Dto {
    export class Country {
        public id: string;
        public isoCode: string;
    }

    export interface ICountryLocalised {
        country: Country;
        locale: Dto.Locale;
        value: string;
    }

    export class CountryLocalised implements ICountryLocalised{
        public country: Country;
        public locale: Dto.Locale;
        public value: string;
    }
}

File: src/KnightFrank.Antares.Dal/Model/Resource/CountryLocalised.cs
namespace KnightFrank.Antares.Dal.Model.Resource
{
    using System;

    using KnightFrank.Antares.Dal.Model.Common;

    public class CountryLocalised : BaseEntity, ILocaled
    {
		public Guid CountryId { get; set; }

		public virtual Country Country { get; set; }

		public Guid LocaleId { get; set; }

		public virtual Locale Locale { get; set; }

		public string Value { get; set; }
	}
}